app/assets/javascripts/shops/order.js

```diff
@@ -0,0 +1,26 @@
+$(function() {
+  $('#btn-update-order').click(function() {
+    let order_status = $(this).parent().find('#order-status');
```

Comment:
nên đặt là $order_status e nhá. để biết đó là 1 element

Reply:
E vừa mới tìm hiểu kiểu chị bảo, trước giờ e lại toàn dùng let var